### PR TITLE
update: new screenshots and UI changes for IDEA 2021.3

### PR DESCRIPTION
**Pull request by danil-pavlov**
* KT-49900 info on Regular framework / CocoaPods setting

### CODE REVIEW

File: docs/topics/compiler-reference.md [hunk 1]
```diff
@@ -14,14 +14,14 @@ in the [Working with command-line compiler](command-line.md) tutorial.
 Kotlin compilers have a number of options for tailoring the compiling process.
 Compiler options for different targets are listed on this page together with a description of each one.
 
-There are several ways to set the compiler options and their values (_compiler arguments_):
-- In IntelliJ IDEA, write in the compiler arguments in the __Additional command-line parameters__ text box in 
-__Settings | Build, Execution, Deployment | Compilers | Kotlin Compiler__
-- If you're using Gradle, specify the compiler arguments in the `kotlinOptions` property of the Kotlin compilation task.
+There are several ways to set the compiler options and their values (*compiler arguments*):
```
Comment (koshachy) [inline docs/topics/compiler-reference.md:17]:
```suggestion
There are several ways to set the compiler options and their values (_compiler arguments_):
```

Comment (koshachy) [reply docs/topics/compiler-reference.md:17]: Italic uses underscores, bold uses asterisks :)

File: docs/topics/js/dev-server-continuous-compilation.md [hunk 2]
```diff
@@ -8,20 +8,20 @@ in _continuous_ mode:
 ./gradlew run --continuous
 ```
 
-If you are working in IntelliJ IDEA, you can pass the same flag via the _run configuration_. After running the Gradle
+If you are working in IntelliJ IDEA, you can pass the same flag via the *run configuration*. After running the Gradle
```
Comment (koshachy) [inline docs/topics/js/dev-server-continuous-compilation.md:11]:
```suggestion
If you are working in IntelliJ IDEA, you can pass the same flag via the _run configuration_. After running the Gradle
```

File: docs/topics/js/js-debugging.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ on it. Refer to your browser's documentation for more details.
 
 To debug Kotlin/JS in the browser:
 
-1. Run the project by calling one of the available `*Run` Gradle tasks, for example, `browserDevelopmentRun` or
+1. Run the project by calling one of the available `run` Gradle tasks, for example, `browserDevelopmentRun` or
```
Comment (koshachy) [inline docs/topics/js/js-debugging.md:16]: There was an asterisk sign to show that the Command doesn't start with the `run` word.
Do you think we should omit that?

File: docs/topics/js/js-debugging.md [hunk 4]
```diff
@@ -39,31 +39,31 @@ when the script is loaded for the first time.
 
 [IntelliJ IDEA Ultimate](https://www.jetbrains.com/idea/) provides a powerful set of tools for debugging code during development.
 
-For debugging Kotlin/JS in IntelliJ IDEA, you'll need a **Debug JavaScript** debug configuration. To add such a configuration:
+For debugging Kotlin/JS in IntelliJ IDEA, you'll need a **JavaScript Debug** configuration. To add this debug configuration:
```
Comment (koshachy) [inline docs/topics/js/js-debugging.md:42]: Nothing wrong with "such a". The wording is OK.

File: docs/topics/jvm/mixing-java-kotlin-intellij.md [hunk 5]
```diff
@@ -53,24 +53,24 @@ println(customer.placeOrder())
...
+![Bundling Kotlin runtime](bundling-kotlin-option.png){width=600}
 
-You can also open the Kotlin runtime configuration manually from __Tools \| Kotlin \| Configure Kotlin in Project__.
+You can also open the Kotlin runtime configuration manually from **Tools** | **Kotlin** | **Configure Kotlin in Project**.
 
 ## Converting an existing Java file to Kotlin with J2K
 
-The Kotlin plugin also bundles a Java to Kotlin converter (_J2K_) that automatically converts Java files to Kotlin.
-To use J2K on a file, click __Convert Java File to Kotlin File__ in its context menu or in the __Code__ menu of IntelliJ IDEA.
+The Kotlin plugin also bundles a Java to Kotlin converter (*J2K*) that automatically converts Java files to Kotlin.
```
Comment (koshachy) [inline docs/topics/jvm/mixing-java-kotlin-intellij.md:70]:
```suggestion
The Kotlin plugin also bundles a Java to Kotlin converter (_J2K_) that automatically converts Java files to Kotlin.
```

File: docs/topics/run-code-snippets.md [hunk 6]
```diff
@@ -47,28 +47,28 @@ We can use classes or functions from a Kotlin project, in our scratches and work
...
+To evaluate each particular expression in a scratch or a worksheet, run it with **Use REPL** selected. The code will be 
 executed the same way as in [REPL](#repl): the code lines will run sequentially, providing results of each call. 
 You can later refer to the results by the names `res*` shown in the corresponding lines.
 
 ![Scratch REPL](scratch-repl.png){width=700}
 
 ## REPL
 
-_REPL_ (_Read-Eval-Print-Loop_) is a tool for running Kotlin code interactively. REPL lets you evaluate expressions and 
+*REPL* (*Read-Eval-Print-Loop*) is a tool for running Kotlin code interactively. REPL lets you evaluate expressions and
```
Comment (koshachy) [inline docs/topics/run-code-snippets.md:68]:
```suggestion
_REPL_ (_Read-Eval-Print-Loop_) is a tool for running Kotlin code interactively. REPL lets you evaluate expressions and 
```

File: .gitmodules [hunk 7]
```diff
@@ -1,3 +1,4 @@
 [submodule "kotlinx.coroutines"]
 	path = kotlinx.coroutines
 	url = https://github.com/Kotlin/kotlinx.coroutines
+	branch = idea-update
```
Comment (koshachy) [inline .gitmodules:4]: Don't forget to change it back ro master

**Review by koshachy — COMMENTED** [review]
Added some formatting suggestions. You can easily add them to the bunch and create a commit via the GitHub UI (**Files changed** tab).

File: docs/topics/js/js-debugging.md (hunk 3 above)
Comment (danil-pavlov) [reply docs/topics/js/js-debugging.md:16]: It was confusing to me, I thought it's a formatting mistake

File: docs/topics/js/js-debugging.md (hunk 4 above)
Comment (danil-pavlov) [inline docs/topics/js/js-debugging.md:42]:
```suggestion
For debugging Kotlin/JS in IntelliJ IDEA, you'll need a **JavaScript Debug** configuration. To add such a debug configuration:
```

File: docs/topics/js/js-debugging.md (hunk 3 above)
Comment (koshachy) [reply docs/topics/js/js-debugging.md:16]: Maybe we could think how not to confuse to our readers...

**Pull request merged**